multi-arch/samples.md

```diff
@@ -0,0 +1,158 @@
+to achieve the desired results within the context of how multi-arch is being achieved for 
+the entire product.
+
+Alex Crawford and Trevor King have indicated that 
+ - clusters would remain homogenous at least through 4.5/4.6 at the time of this writing
+ - use of golang's runtime.GOARCH is sufficient for determining one's architecture
+
+That said, the samples operator config object currently has a []string array defined "Architectures"
+in anticipation of heterogenous clusters.  If that requirement ever gets into plan, the current understanding
+after talking to Alex and Trevor are
+ - some sort of list of architectures in a cluster would be provided in the infrastructure config
+ - k8s styled selectors of some sort will be provided/utilized to pin pod leveraging non-x86 images to the required nodes; concerns for this pinning thus would not fall on the samples operator
+
+## Motivation
+
+Multi-arch has been declared a key initiative for v4.x.
+
+### Goals
+
+x86 samples are not installed in a cluster with no x86 nodes.  Nor are any future non-x86 samples installed in a cluster with only x86 nodes.
+
+### Non-Goals
+
+We do not take on porting existing samples to non-x86 architectures.
+
+## Proposal
+
+The samples operator config object already supports the specification of multiple 
+architectures.
+
+It currently requires that only a single architecture is specified, and that 
+it is x86_64.
+
+In conjunction, the import of content from https://github.com/openshift/library supports 
+downloading multiple architectures, and segregating multiple architectures.  And 
+the tagging in https://github.com/openshift/library allows providers to designate which
+samples go to which platforms.
+
+So enabling the above to allow PPC and Z content if it is available is item one.
+
+But as previously stated, there is currently no such content, so the samples operator
+should do nothing when it comes up in Managed state, and report Available=true, Progressing=false,
+Degraged=false.
+
+We also need to ensure that marking the samples operator as Unmanaged or Removed 
+
+There are some interesting details and caveats that will be discussed in the 
+implementation details section.
+
+### User Stories [optional]
+
+#### Story 1
+
+#### Story 2
+
+### Implementation Details/Notes/Constraints [optional]
+
+In a somewhat related note, there have been a lot of side conversations regarding the Jenkins imagestream
+in particular, as the openshift/jenkins images are included in the 4.1/4.2 payloads.
+
+Clayton weighed in with Ben Parees and they concluded that unless/until the openshift/jenkins images 
+are fully removed from the install payload (i.e. the devtools team, say as part of the Jenkins operator work, 
+somehow facilitates the removal of the openshift/jenkins images from the payload), the x86 openshift/jenkins 
+images will be utilized as part of building the non-x86 install payload.
+
+### Risks and Mitigations
+
```

Comment:
Should we call out here or ack that the solution needs to not prevent changes for mixed clusters in the future or how this is expected to evolve?  Maybe overkill since it was already mentioned, just seems like a constraint.

Reply:
added